.changeset/great-scissors-pretend.md

```diff
@@ -0,0 +1,7 @@
+---
+"create-cloudflare": minor
+"playground-preview-worker": patch
```

Comment:
how come this is diff than the other two packages?

Reply:
My logic was that this was a feature for both C3 (adding an explicit schema to generated templates), and Wrangler (generated the schema), but only a patch for playground-preview-worker, since we're just updating the config file to include a schema comment